### DISCUSSION TITLE
Localization for Chinese Policy

### DISCUSSION THREAD

**Discussion by Molyleaf** [requarks/wiki#4697] [Help / Questions] [answered]
China authority requires a government-record number to be placed on the foot of the homepage and the login screen.Can you offer a input box to input additional word or custom html code shown in the footer panel.

**Comment by erowz** [comment]
绷不住了

**Comment by scienceasdf** [comment, marked as answer]
My solution may not be elegant, but sufficient to this problem.

In your administrator panel, go to theme settings, and embed the following lines to the HTML code:
```html
<script type="text/javascript">
window.onload = function(){
  var footerInfo = document.createElement("span");
  footerInfo.innerHTML = "<br>蚌埠住了";
  footerNode = document.getElementsByTagName("footer")[0];
  footerNode.firstChild.appendChild(footerInfo);
  footerNode.firstChild.style["text-align"] = 'center';
};
</script>
```
Save it and you will get what you want. Another way is to modify the `nav-footer.vue` source file and rebuild the server.

我的解法比较粗暴，不过可以直接解决你的问题。

在管理员面板中，进入主题设置菜单，在“正文HTML注入”中加入上面的代码并保存就OK了。你也可以直接修改vue的源码，然后用yarn重新构建服务器。

**Reply by Moebyte** [reply]
请问是把这一段代码加进`nav-footer.vue`吗？

**Reply by 573077632** [reply]
您的管理员面板中，转到主题设置，然后将以下行嵌入 HTML 代码：

**Reply by lu-zhibin** [reply]
请问如果要给备案号加上链接呢？可以实现吗？

**Reply by zcorn2017** [reply]
同问，似乎不行，改成链接好像就不显示了

> 请问如果要给备案号加上链接呢？可以实现吗？

**Reply by SUNNY-CT** [reply]
> 请问如果要给备案号加上链接呢？可以实现吗？

可以啊  页脚支持MARKDOWN的编辑方式 [备案号](连接）就可以了

**Comment by NGPixel** [comment]
An additional footer text field has been added to the upcoming v3 release for such scenarios:

<img width="738" alt="image" src="https://user-images.githubusercontent.com/15522395/174522593-0de89590-0d89-435e-bc50-3270accee07b.png">

**Reply by Moebyte** [reply]
It sounds great!
